runtime/gc_modron_startup/mmhelpers.cpp

```diff
@@ -322,7 +327,39 @@ j9gc_modron_getConfigurationValueForKey(J9JavaVM *javaVM, UDATA key, void *value
 		*((UDATA *)value) = extensions->compressObjectReferences();
 		keyFound = TRUE;
 		break;
-		
+	case j9gc_modron_configuration_heapBase:
```

Comment:
We don't need to add a key for the heap base, as the JIT currently receives this information from the `J9VMThread`. [See](https://github.com/eclipse/openj9/blob/master/runtime/gc_modron_startup/mminit.cpp#L200).

Reply:
Fixed this.